tests/routes/cbauthentication_tests/cb_password_reset.test.js

```diff
@@ -1,18 +1,25 @@
 const test = require('tape');
 const request = require('supertest');
+const nock = require('nock');
 const createApp = require('../../../react-backend/app');
-const dbConnection = require('../../../react-backend/database/dbConnection');
-const rebuild = require('../../../react-backend/database/database_rebuild');
+const refreshDB = require('../../../db/scripts/refresh');
 const { getConfig } = require('../../../config');
 
 const config = getConfig(process.env.NODE_ENV);
 
 
 test('POST api/cb/pwd/reset | token creation successful', async (t) => {
-  await rebuild();
+  await refreshDB();
   const app = createApp(config);
-
+  const dbConnection = app.get('client:psql');
   const successPayload = { formEmail: 'findmyfroggy@frogfinders.com' };
+
+  nock('https://api.postmarkapp.com')
+    .post('/email/withTemplate')
+    .reply(200, {})
+    .post('/email/batch')
+    .reply(200, {})
+
```

Comment:
So nock will intercept any post requests with those two routes to the postmark api; and if we get 200's back we know that it will at least successfully send the request?

Reply:
`nock` interceptors work by [default for one matched request each](https://github.com/node-nock/nock#read-this---about-interceptors), so this snippet will set up one interception on `POST /email/batch` and one on `POST /email/withTemplate`. Both will "respond" with a `200` status code and the response body will be an empty object.